core/include/detray/propagator/propagator.hpp

```diff
@@ -150,9 +150,9 @@ struct propagator {
     ///
     /// @note If the return value of this function is true, a propagation step
     /// can be taken afterwards.
+    template <typename actor_states_t>
```

Comment:
Hmm, does this mean we can now support having multiple valid actor states for one given propagation configuration?

Reply:
Yes. For example, permutations of the actor states are allowed (forward/backward propagation) and empty states are left out (they are not useful anyway). In a follow up PR, I put a concept here to make this a bit safer:
```
    template <typename actor_states_t>
    requires concepts::is_state_of<actor_states_t, actor_chain_type>
```

Reply:
Okay, makes sense. Thanks!